yaksh/models.py

```diff
@@ -280,10 +298,46 @@ def set_marks(self, marks):
     def __unicode__(self):
         return self.answer
 
+
 ###############################################################################
 class QuizManager(models.Manager):
     def get_active_quizzes(self):
-        return self.filter(active=True)
+        return self.filter(active=True, is_trial=False)
+
+    def create_trial_quiz(self, trial_course, user):
+        """Creates a trial quiz for testing questions"""
+        trial_quiz = self.create(course=trial_course,
+                                 duration=1000,
+                                 description="trial_quiz",
+                                 is_trial=True,
+                                 time_between_attempts=0
+                                 )
+        return trial_quiz
+
+    def create_trial_from_quiz(self, original_quiz_id, user, mode):
```

Comment:
Can you make `mode`, `godmode=False` as it appears you only have two modes user mode and god mode.   This way instead of a string argument it is a boolean which defaults to False.